proto/cosmos/tx/v1beta1/service.proto

```diff
@@ -45,6 +50,36 @@ message GetTxsEventResponse {
   cosmos.base.query.v1beta1.PageResponse pagination = 3;
 }
 
+// BroadcastTxRequest is the request type for the Service.BroadcastTxRequest
+// RPC method.
+message BroadcastTxRequest {
+  // tx_raw is the raw transaction.
+  cosmos.tx.v1beta1.TxRaw tx_raw = 1;
```

Comment:
Just noting that I don't have a strong opinion about this being TxRaw if you think bytes is better @amaurymartiny

Reply:
Yeah and also the more I think about it `bytes` is actually better because we don't want to re-encode which could change the hash. The client should be able to know the hash of the tx that gets broadcast. So sorry please revert to `bytes`

Reply:
I switched back to bytes